app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script/api_create.rb

```diff
@@ -1,14 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript
   module ApiCreate
-    def create_in_provider(manager_id, params)
+    def create_in_provider(manager_id, params, inline = false)
       manager = ExtManagementSystem.find(manager_id)
       job_template = manager.with_provider_connection do |connection|
         connection.api.job_templates.create!(params)
       end
 
       # Get the record in our database
       # TODO: This needs to be targeted refresh so it doesn't take too long
-      EmsRefresh.queue_refresh(manager, nil, true) if !manager.missing_credentials? && manager.authentication_status_ok?
+      if inline && !manager.missing_credentials? && manager.authentication_status_ok?
```

Comment:
@syncrou Do we need to check `missing_credentials?` here as well as `authentication_status_ok?`.  I would think `status_ok?` would be enough.